src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/TypeLoader/ExternalReferencesTable.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime;
+
+namespace Internal.Runtime.TypeLoader
+{
+    //
+    // Temporary duplicate of ExternalReferencesTable in the TypeLoader
```

Comment:
Is the expectation we would be folding S.P.TypeLoader into CoreLib as well?

Reply:
I would like to fold Reflection that also requires ExternalReferencesTable.

TypeLoader can probably stay as is as long as it does not get into way. There is not much to share between the managed type system and unmanaged type system at this point.

Reply:
Is is possible to move ExternalReferencesTable into Common so that we don't have two copies?